Assets/Scripts/Export/ExportFbx.cs

```diff
@@ -194,7 +194,7 @@ class ExportMesh {
       try {
         WriteObjectsAndConnections2(G);
         G.m_exporter.Export(G.m_scene);
-      } catch (InvalidOperationException e) {
+      } catch (GlbError e) {
```

Comment:
GlbError for FBX export?

Reply:
Yeah. One of the problems is that I did this way back as a quick experiment to see if I could get export working. 

I suspect that might be my past self going "let's put an exception in here that won't happen as it's quicker than commenting out the whole block". Exactly the kind of sloppy code I was hoping this review would catch ;-)

I'll remove that change and see what I was actually aiming to achieve here.

Have you taken a look at my shader edits? There's probably some bad things lurking there as well.

Reply:
Fixed